awx/ui_next/src/screens/ExecutionEnvironment/shared/ExecutionEnvironmentForm.jsx

```diff
@@ -124,8 +126,15 @@ function ExecutionEnvironmentFormFields({
 
       <CredentialLookup
         label={i18n._(t`Registry credential`)}
+        credentialTypeKind="registry"
```

Comment:
:+1: Nice. Didn't know we already had a param for this!

Reply:
We do also have a parameter to filter by credential namespace. Credential lookup is bit more complex due to those flags AFAIK.